collector/container/konflux.Dockerfile

```
@@ -130,6 +130,7 @@ LABEL \
     io.k8s.description="This image supports runtime data collection for Red Hat Advanced Cluster Security for Kubernetes" \
     io.openshift.tags="rhacs,collector,stackrox" \
     maintainer="Red Hat, Inc." \
+    # We must set source-location to provide it to our custom Snapshot.
```

Question: can @msugakov find something to comment on in 1-line-change PR?
Answer: was it a rhetorical question?

While this comment is good, I suggest to point curious minds to where it happens so that anyone who decides to change things can trace without having to go through commit and PR history.

Something like this:

```suggestion
    # Custom Snapshot creation in `operator-bundle-pipeline` depends on source-location label to be set correctly.
```